receiver/elasticsearchreceiver/go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/elasti
 
 go 1.17
 
+replace github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest => ../../internal/scrapertest
```

Comment:
Typically replace directives are listed at the end of the file. Was this autogenerated?

Reply:
I did that manually; Moved to the bottom.